pkgs/tools/package-management/nix/default.nix

```diff
@@ -39,23 +39,23 @@ let
     inherit name src;
     version = lib.getVersion name;
 
-    is112 = lib.versionAtLeast version "1.12pre";
+    is20 = lib.versionAtLeast version "2.0pre";
```

Comment:
I'm not sure if this needs to be changed. 2.0 is bigger than 1.12 right?

Reply:
This is a commit I merely cherry-picked from master :)